src/main/java/dev/jbang/catalog/ImplicitCatalogRef.java

```diff
@@ -52,7 +52,11 @@ public static ImplicitCatalogRef parse(String name) {
 		if (names.length > 3) {
 			return null;
 		}
-		String org = names[0];
+		String org = null;
+		if (names.length > 2) {
+			org = names[0];
+		}
+		org = "jbangapp"; // TODO: should global default be controllable by config?
```

Comment:
I think we should just make it a constant, just like `Catalog.JBANG_CATALOG_JSON`